.github/workflows/bootstrap-image.yml

```diff
@@ -24,19 +24,24 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: login to Artifactory
+        id: login
+        run: docker login ${{ env.ARTIFACTORY }} -u ${{ env.ARTIFACTORY_USER }} -p ${{ env.ARTIFACTORY_API_KEY }}
```

Comment:
Maybe keep the `docker login` as it was with the `--password-stdin` method.

Reply:
updated